tests/conftest.py

```diff
@@ -220,17 +220,6 @@ def set_unstreamed_values(value):
     return completion_mock
 
 
-@pytest.fixture(scope="function")
-def mock_call_embedding_api(mocker):
-    embedding_mock = mocker.patch.object(LlmApiHandler, "call_embedding_api")
-
-    def set_embedding_values(value):
-        embedding_mock.return_value = value
-
-    embedding_mock.set_embedding_values = set_embedding_values
-    return embedding_mock
-
-
```

Comment:
I've removed this and embeddings_test.py, because the behavior we were actually testing (properly measuring similarity) is now handled by chroma.

Reply:
We're still gonna need this mock, because otherwise we can never use call_embedding_api in tests (since it has an api_guard on it)

Reply:
As it's setup we wouldn't ever need to, so that func could be removed as well.

But in light of your comment below this will change